packages/react-theme-provider/src/getTokens.ts

```diff
@@ -63,12 +63,12 @@ export function getTokens(theme: Theme): Tokens {
       },
 
       button: {
-        fontWeight: fonts?.medium?.fontWeight,
+        fontWeight: 600, // fonts?.medium?.fontWeight,
         fontSize: fonts?.medium?.fontSize,
         fontFamily: fonts?.medium?.fontFamily,
         iconSize: fonts?.mediumPlus?.fontSize,
         borderRadius: effects?.roundedCorner2,
-        focusColor: palette?.neutralSecondary,
+        focusColor: palette?.black,
```

Comment:
are you sure? in V7 we have are using `neutralSecondary` for the outline color. however the focus border style has changed in v8, so maybe this is correct

Reply:
Pretty sure this is correct. `neutralSecondary` is not dark enough. I can confirm with design.